form.php

```diff
@@ -0,0 +1,46 @@
+<?php
+require_once 'functions.php';
+
+?>
+
+<!--write the form with inputs-->
+<html>
+<head>
+    <link rel="shortcut icon" href="images/favicon.ico"/>
+    <title>Add to the Cheese Collection</title>
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link href="https://fonts.googleapis.com/css?family=Indie+Flower&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
```

Comment:
maybe a button for them to go back to the collections page if they didn't want to add in the end

Reply:
ops that should be below the form, my bad